index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
```

Comment:
I'd prefer `outputLatency` as it matches to `outputTimeStamp` in the latest patch. IMO `outputLatency`  should remain at `AudioDestinationNode` as this interface is representing an output device, so it's a natural place to contain the attribute representing output device latency. wdyt?